app/script/entity/Conversation.js

```diff
@@ -277,9 +277,10 @@ z.entity.Conversation = class Conversation {
    * @note This will only increment timestamps
    * @param {string|number} timestamp - Timestamp to be set
    * @param {z.conversation.TIMESTAMP_TYPE} type - Type of timestamp to be updated
+   * @param {boolean} allowDecrement - Allow setting a timestamp older than the current timestamp
    * @returns {boolean|number} Timestamp value which can be 'false' (boolean) if there is no timestamp
    */
-  set_timestamp(timestamp, type) {
+  set_timestamp(timestamp, type, allowDecrement = false) {
```

Comment:
I am not fully sure on the naming. Wdyt about `forceUpdate` maybe?

Reply:
`forceUpdate` LGTM